src/pages/index.vue

```diff
@@ -1,45 +1,213 @@
+            </nuxt-link>
+          </div>
+          <div class="title is-5" v-else>
+            {{ article.title }}
+          </div>
+        </div>
+        <nav class="level is-mobile">
+          <!-- なぜかこれ入れないと右に寄らない． -->
+          <div class="level-left"></div>
+          <div class="level-right">
+            <div class="level-item">
+              <p>year: {{ article.year }}</p>
+            </div>
+            <div class="level-item">
+              <p>citation: {{ article.citation }}</p>
+            </div>
+          </div>
+        </nav>
+      </div>
     </div>
-    <button @click="getUsername">get</button>
   </section>
 </template>
 
 <script>
-import Card from '~/components/Card'
+import axios from 'axios'
 import firebase from '~/plugins/firebaseSettings'
-
 export default {
-  name: 'HomePage',
-
-  components: {
-    Card
+  name: 'SearchPage',
+  components: {},
+  data() {
+    return {
+      query: '',
+      qarray: [],
+      info: {
+        word: '',
+        cname: '',
+        year: ''
+      },
+      params: {
+        model: 'latest',
+        count: 20,
+        offset: '0',
+        orderby: 'Y:desc',
+        attributes: 'Ti,Y,CC'
+      },
+      articles: []
+    }
   },
   methods: {
-    async getUsername() {
-      const user = await firebase.auth().currentUser
-      console.log(user)
+    async searchApi() {
+      this.articles = []
+      this.query = ''
+      this.qarray = []
+      const db = firebase.firestore()
+
+      // create query
+      const spword = this.info.word
+        .toLowerCase()
+        .replace(/\W/g, ' ')
+        .replace(/\s{2}/g, ' ')
+        .split(' ')
+
+      spword.forEach((el) => {
+        if (el !== '') {
+          this.qarray.push("W='" + el + "'")
+        }
+      })
+
+      if (this.info.cname !== '') {
+        this.qarray.push(
+          "Or(Composite(C.CN='" +
+            this.info.cname.toLowerCase() +
+            "'),Composite(J.JN='" +
+            this.info.cname.toLowerCase() +
+            "'))"
+        )
+      }
+
+      if (this.info.year !== '') {
+        this.qarray.push('Y=' + this.info.year)
+      }
+
+      if (this.qarray.length === 0) {
+        // error handling
+        return
+      } else if (this.qarray.length === 1) {
+        this.query = this.qarray[0]
+      } else {
+        this.query = this.query + 'And('
+        this.qarray.forEach((el) => {
+          this.query = this.query + el + ','
+        })
+        this.query = this.query.slice(0, -1)
+        this.query = this.query + ')'
+      }
+
+      console.log(this.query)
+
+      // created
+
+      // create search options
+      const options = require('querystring').stringify(this.params)
+      const url =
+        process.env.academicApiUrl +
+        'evaluate?expr=' +
+        this.query +
+        '&' +
+        options
+      // 非常に危なかったので消した．
+      // console.log(process.env.academicApiHeader)
+
+      const result = await axios.get(url, {
```

Comment:
```suggestion
      const result = await this.$axios.get(url, {
```
こうやって書くことで `import` が書かなくて良くなります！
動いているのでそのままで大丈夫ですが、ちょっとしたtipsです